src/main.py

```diff
@@ -92,10 +114,16 @@ def start_application(args=None):
 
 if __name__ == "__main__":
     # Check the python version
-    if not sys.version_info.major >= 3 and sys.version_info.minor >= 6:
+    if (
```

Comment:
Again, this isn't sitting right with me. I think the `not` should perhaps be pulled outside of the parentheses?

```
if not (
    X
    and Y
)
```